src/workerd/api/node/tests/fs-access-test.js

```diff
@@ -291,4 +296,45 @@ export const existsCallbackTest = {
   },
 };
 
+export const stdioFds = {
+  test() {
+    let stat1 = fstatSync(0);
+    let stat2 = fstatSync(1);
+    let stat3 = fstatSync(2);
+    ok(stat1.isCharacterDevice());
+    ok(stat2.isCharacterDevice());
+    ok(stat3.isCharacterDevice());
```

Comment:
Shouldn't this be `isFIFO()`?

Reply:
I thought so originally also.. but 
![image](https://github.com/user-attachments/assets/003da5b1-dee9-480f-96ae-c2716ca85cbf)

Reply:
You need to explicitly do the test with a pipe - `node test.js | cat`. The results vary depending on REPL / `node test.js` / `node test.js > out` and `node test.js | cat`. Also been doing a bunch of these investigations today!

Reply:
Right, I can get behind the default being `isCharacterDevice()`.